lib/physiqual/data_services/fitbit_service.rb

```diff
@@ -86,9 +88,12 @@ def process_intraday_entries(entries, date)
         return {} if entries.nil?
         entries.each do |entry|
           value = entry['value']
-          value = convert_to_int_if_needed(value)
-          time_and_date = Time.parse("#{date} #{entry['time']}")
-          result << { date_time_field => time_and_date, values_field => [value] }
+          current_value = [convert_to_int_if_needed(value)]
+          endd = Time.parse("#{date} #{entry['time']}").in_time_zone
+          start = endd - 1.minute
```

Comment:
Wat is hier de logica achter? Wat is de precisie van `entry['time']` hier? Minuten of seconden?